docs/content/concepts/automation.mdx

```diff
@@ -0,0 +1,90 @@
+
+---
+
+## Prerequisites
+
+Before continuing, you should be familiar with:
+
+- [Software-defined Assets][assets]
+- [Jobs][jobs] (_optional_)
+- [Ops][ops] (_optional; advanced_)
+
+---
+
+## Available methods
+
+In this section, we'll touch on each of the automation methods currently supported by Dagster. After that we'll discuss what to think about when [selecting a method](#selecting-a-method).
+
+### Schedules
+
+Schedules kick off [jobs][jobs] at fixed time intervals. Using schedules, you can start jobs at specific times, such as Mondays at 9:00 AM. Jobs triggered by schedules can contain a subset of [assets][assets] or [ops][ops]. Refer to the [Schedules documentation][schedules] to learn more.
+
+### Sensors
+
+To run a job or materialize an asset in response to specific events, you can use sensors. Sensors continuously check and execute logic to know whether to kick off a run. They are commonly used for situations where you want to materialize an asset after something else happens, such as:
+
+- A new file arrives in a specific location, such as Amazon S3
+- Another asset elsewhere has materialized
+- An external system frees up a worker slot
+
+You can also use sensors to act on the status of a job run. Refer to the [Sensors documentation][sensors] to learn more.
+
+### Auto-materialize policies <Experimental />
+
+Unlike schedules, Auto-materialize Policies (AMP) don't run at a specific time. Instead, an AMP allows you to describe the conditions under which assets should be materialized or skipped. An AMP, for example, could kick off a run for a downstream asset immediately after upstream changes occur.
+
+AMPs are declared on an asset-by-asset basis, but can be applied to multiple assets at once. Refer to the [Auto-materializing Assets documentation][auto-materialize-policies] to learn more.
+
+### Asset Sensors <Experimental />
+
+Asset sensors trigger jobs when a specified asset is materialized. Using asset sensors, you can instigate runs across jobs and code locations and keep downstream assets up-to-date with ease.
+
+Refer to the [Asset Sensor documentation][asset-sensors] to learn more.
+
+---
+
+## Selecting a method
+
+Before you dive into automating your pipelines, you should think about:
+
+- Is my pipeline made up of assets, ops, graphs, or some of everything?
+- How often does the data need to be refreshed?
+- Is the data partitioned, and do old records require updates?
+- Should updates occur in batches? Or should updates start when specific events occur?
+
+The following cheatsheet contains high-level details about each of the automation methods we covered, along with an example of when each one would be used.
+
+| Name                                                                  | How it works                                                                                                      | Works with          | Example                                                              |
```

Comment:
How can we work partitions into this? Should we?

Reply:
I don't know if partitions fits here, I think the primary rubric is whether or not you are working from jobs or not.
I wonder if maybe we can be a little more prescriptive here?

If you are triggering based off an event: use sensors
If you are using jobs: use schedules
If you are not using jobs and are comfortable with experimental APIs, and want a declarative approach: use AMP